root/etc/nginx/nginx.conf.template

```diff
@@ -0,0 +1,46 @@
+daemon off;
+
+http {
+    include /etc/nginx/proxy.conf;
+    include /etc/nginx/mime.types;
+
+    server {
+	    client_body_temp_path /tmp/nginx_client_temp 1 2;
+        listen 8080 default_server;
+        resolver ${DNS_SERVER};
+        access_log /dev/stdout;
+
+        ## Expose core platform millicore and security endpoints that are used to perform 
+        ## Mobile app init (retrieving the actual app url basing on connection tag)
+        location ~* ^/core/box/srv/1.1/(.*) {
+           proxy_pass ${CORE_URL}/box/srv/1.1/$1$is_args$args;
```

Comment:
@wtrocki Based on white board we said this would be BaseHost ? I don't think we need each of the different env vars here?